serenata_toolbox/datasets/downloader.py

```diff
@@ -38,7 +38,8 @@ class Downloader:
         '2017-03-15-reimbursements.xz',
         '2017-03-20-purchase-suppliers.xz',
         '2017-04-27-companies-no-geolocation.xz',
```

Comment:
I'm not sure. If we think about versioning, this should be there. Maybe someone somewhere in the world is running an analysis using it. But I do agree it's a quiet odd possibility. So… being strict, no this should not be deleted… but being pragmatic, yes, let's get rid of that.